extensions/cache/src/main/resources/benchmark-redis.xml

```diff
@@ -0,0 +1,69 @@
+<benchmark xmlns="urn:radargun:benchmark:3.0">
+
+   <master bindAddress="${master.address:127.0.0.1}" port="${master.port:2103}"/>
+
+   <clusters>
+      <cluster size="8">
```

Comment:
Actually that's a limitation of Jedis that I didn't mention - they have a separate API for singlenode/cluster usecases of Redis, and I'm using the cluster one, so the plugin won't work with anything less than 3 server nodes (3 masters, no slaves = no replicas).
I never ran the tests.sh script, didn't even know it existed.